### PR TITLE
Add question models, controller and tests

### LINKED ISSUE #1: Weekly Scrum - Mar. 5

### PR DESCRIPTION
**Pull request by frankliu197**
Unit test ISSUE. Order not enforced even with the @Order annotation.
Temporary Solution: Merge the tests into one

### CODE REVIEW

File: bin/README.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# Mini-SurveyMonkey-4806
+A simple knock-off of Survey Monkey for the SYSC 4806 Group Project
```
Comment (Binyamin-Brion) [inline bin/README.md]: Should this be in the bin folder?

**Comment by Binyamin-Brion** [comment]
Was wondering two things:
* Is there a reason for committing class files?
* Doesn't maven typically create executables in folder called target, not bin?

**Comment by frankliu197** [comment]
Yes forgot to add bin to .gitignore. I updated it

**Comment by doodlehead** [comment]
Can you put some more comments? Just describe/label what's temporary/incomplete and high level comments for classes. 

Also maybe add some test for REST endpoints using the MockMVC thing?

**Comment by frankliu197** [comment]
Added basic comments. I believe Integration tests should be done when multiple controllers are coded.

**Comment by Binyamin-Brion** [comment]
Code structure looks fine, though going forward more comments is worth considering. Can merge,

**Review by aeiou80 — COMMENTED** [review]
Looks good!

**Pull request merged**